frontend/app/src/util/AppNavigation.ts

```diff
@@ -243,7 +247,7 @@ export class StrategyV2 {
       document.title = title
       this.appNav.hostCommunicationMgr.sendMessageToHost({
         type: "SET_PAGE_TITLE",
-        title,
+        title: currentPage.pageName ?? "",
```

Comment:
If I read this code correctly, it looks like previously we set `Streamlit` as the title if `pageName` was empty [here](https://github.com/streamlit/streamlit/blob/b7c39dfed3cc7fc7d0e05a2fa3a589fa7ccc0f74/frontend/app/src/util/AppNavigation.ts#L47-L53), whereas here it looks like we set it to an empty string. Is that we want?

Reply:
Yes. So this was a bug in the prototype because Community Cloud [adds the Streamlit in the title](https://github.com/streamlit/s4t/blob/59e3f4533776d8adaeaae4436d39facfa4686080/web/src/App.tsx#L240) for us. We don't need to set that here, so the best we can do